apps/condo/domains/common/constants/featureflags.js

```diff
@@ -15,6 +15,7 @@ const ENABLE_DISCOVER_SERVICE_CONSUMERS = 'enable-discover-service-consumers'
 const MOBILE_FEATURE_CONFIGURATION = 'mobile-feature-configuration'
 const TICKET_SUBMITTING_FORM_RESIDENT_MOBILE_APP = 'ticket-submitting-from-resident-mobile-app'
 const SUBMIT_ONLY_PROGRESSION_METER_READINGS = 'submit-only-progression-meter-readings'
+const SEND_BILLING_RECEIPTS_ON_PAYDAY_REMAINDER_TASK = 'send-billing-receipts-on-payday-remainder-task'
```

Comment:
Rem**a**inder word has different meaning

```suggestion
const SEND_BILLING_RECEIPTS_ON_PAYDAY_REMINDER_TASK = 'send-billing-receipts-on-payday-reminder-task'
```